session.go

```diff
@@ -471,16 +468,14 @@ func (s *Session) routingKeyInfo(ctx context.Context, stmt string) (*routingKeyI
 		}
 
 		key, _ := inflight.value.(*routingKeyInfo)
-
 		return key, nil
 	}
 
 	// create a new inflight entry while the data is created
-	inflight := new(inflightCachedEntry)
+	inflight = new(inflightCachedEntry)
 	inflight.wg.Add(1)
```

Comment:
I believe that these two lines (above) should be removed as inflight is already created by `execIfMissing`

Reply:
You are right, thx!